src/containers/guest-home-page/login-dialog/LoginDialog.jsx

```diff
@@ -35,15 +35,12 @@ const LoginDialog = () => {
       },
       initialValues: { email: '', password: '' },
       validations: { email }
-    }
-  )
-
+    })
   return (
     <Box sx={styles.root}>
       <Box sx={styles.imgContainer}>
         <Box alt='login' component='img' src={loginImg} sx={styles.img} />
```

Comment:
This will be easier to read if there are more than 2 component properties, each property will be on a separate line

```suggestion
        <Box
          alt='login'
          component='img'
          src={loginImg}
          sx={styles.img}
        />
```